### PR TITLE
Feature/program

### PR DESCRIPTION
**Pull request by mnavyasree1906**
Implemented Page Object Model for Login functionality in OrangeHRM (Folder: 003-loginwithpom)

### CODE REVIEW

File: 003-loginwithpom/src/test/java/tests/LoginTest.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        // Initialize Page Object
+        loginPage = new LoginPage(driver);
+
+        // WebDriverWait
+        int assertionWait = Integer.parseInt(properties.getProperty("assertion.wait.duration"));
+        wait = new WebDriverWait(driver, Duration.ofSeconds(assertionWait));
+    }
+
+    @Test
+    public void testValidLogin() {
```
Comment (linuxramu) [inline 003-loginwithpom/src/test/java/tests/LoginTest.java:47]: create a method for it,    may be link   loginOrangeHRM(username,password)

Comment (linuxramu) [reply 003-loginwithpom/src/test/java/tests/LoginTest.java:47]: once you write method... call that method here !

**Review by linuxramu — CHANGES_REQUESTED** [review]
delete target and test-output

File: 003-loginwithpom/src/test/java/tests/LoginTest.java (hunk 1 above)
Comment (mnavyasree1906) [reply 003-loginwithpom/src/test/java/tests/LoginTest.java:47]: done changes in LoginPage class added login method and also update LoginTest to use login method

**Comment by mnavyasree1906** [comment]
> delete target and test-output

deleted both

**Review by linuxramu — APPROVED** [review]

**Comment by mnavyasree1906** [comment]
Merged to main branch

**Pull request merged**